ecosystem/sep-0023.md

```diff
@@ -65,7 +67,10 @@ The following steps transform a binary key into a strkey:
 1.  Start with the appropriate version byte computed by the OR of a key type base value and algorithm selector from the
     tables above.
 
-2.  Append the binary bytes of the key (e.g., 32-bytes for ED25519).
+2.  Append the binary bytes of the key (e.g., 32-bytes for ED25519, 32-bytes for SHA256).
+
+    If we are encoding a claimable balance, the binary bytes of the key has a length of 33-bytes: 1-byte value
```

Comment:
@stanford-scs The claimable balance strkey will not be padded to a multiple of 4. For some other strkeys like the signed payload we did padding, but I don't remember why or if that is truly important. I'm inclined to not pad for smaller strkeys in this case.